src/Compilers/VisualBasic/Portable/Symbols/ConstraintsHelper.vb

```diff
@@ -427,9 +427,59 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
         <Extension()>
         Public Function CheckConstraints(
```

Comment:
>Function [](start = 15, length = 8)

Is anyone checking value returned by this function? Please convert it to a Sub otherwise.